oag/ja-jp/content/topics/access-gateway/cert-workflow-upload.htm

```diff
@@ -341,29 +326,23 @@ <h2><a name="Add_a_new_certificate"></a>証明書を追加する</h2>
+                                                                <li value="4"><span class="uicontrol">Ctrl</span>+<span class="uicontrol">d</span>を押して証明書の内容を保存します。コマンドラインコンソールにより、証明書に関連するキーコンテンツ用に新しいエディターが表示されます。</li>
                                                                 <li value="5">テキストエディタ―でキーファイルを開きます。</li>
-                                                                <li value="6">エディターでキーの内容を選択してコピーします。</li>
-                                                                <li value="7">コマンドラインコンソールに戻り、キーファイルの内容をペーストします。 <br /></li>
-                                                                <li value="8">完了したら、<span class="uicontrol">Ctrl</span> + <span class="uicontrol">d</span>を押してキーの内容を保存します。 </li>
+                                                                <li value="6">キーの内容を選択してコピーします。</li>
+                                                                <li value="7">コマンドラインコンソールに戻り、キーファイルの内容を貼り付けます。</li>
+                                                                <li value="8"><span class="uicontrol">Ctrl</span>+<span class="uicontrol">d</span>を押してキーの内容を保存します。ホスト名と証明書タイプは、証明書に合わせて自動的にプルされます。
+					
+						<p>証明書の更新では、現在の証明書を交換するかどうかの確認が求められます。証明書を更新するには、<span class="uicontrol">y</span>を押してから<span class="uicontrol">Enter</span>を押します。</p></li>
                                                             </ol>
-                                                            <div class="noteOkta">
-                                                                <p class="noteContent">ホスト名と証明書タイプは、証明書に対して自動的にプルされます。</p>
-                                                            </div>
-                                                            <p>
-							証明書の更新では、現在の証明書を置き換えるかどうかの確認が求められます。証明書の更新を続行するには、<span class="uicontrol">y</span>を押し、続けて<span class="uicontrol">Enter</span>を押します。 <br /></p>
                                                         </div>
                                                         <div>
-                                                            <h2><a name="Modify_An_Existing_Certificate"></a>既存の証明書を修正する</h2>
-                                                            <p>既存の証明書を修正する場合、次の3つのオプションが表示されます。</p>
+                                                            <h2><a name="Modify_An_Existing_Certificate"></a>証明書を修正する。</h2>
+                                                            <p>次のいずれかのコマンドを選択します。</p>
                                                             <ul>
-                                                                <li><span class="code">[d] </span> - 証明書を削除する </li>
-                                                                <li><span class="code">[u]</span> - 証明書を更新する</li>
-                                                                <li><span class="code">[x]</span> - 変更せずに戻る </li>
+                                                                <li><span class="uicontrol">d</span>：証明書を削除します。</li>
```

Comment:
UI control labels without the brackets... but these should be `userinput` attrs anyway.